runtime/src/lib.rs

```diff
@@ -189,4 +193,9 @@ pub fn handle_mailbox_commands(drivers: &mut Drivers) {
             }
         }
     }
+
+    loop {
+        caliptra_drivers::report_fw_error_non_fatal(CaliptraError::RUNTIME_SHUTDOWN.into());
+        drivers.mbox.set_status(MboxStatusE::CmdFailure);
+    }
```

Comment:
I don't understand this. Is the idea that once we shutdown, we don't respond to any commands? If that is the case, maybe we should just fatal error.

Reply:
yeah i think we can just report error and enter loop inside the shutdown handler.

Reply:
done